frontends/web/src/locales/en/app.json

```diff
@@ -445,6 +445,7 @@
   },
   "chart": {
     "dataMissing": "Gathering historical data... stay tuned.",
+    "dataOldTimestamps": "Exchange rates for {{name}} are older than expected. Last updated {{timeAgo}}",
```

Comment:
"older than expected" is imo confusing to a user. what is expected?

why does the user even need to see which coin rates are older than 2h? i don't see how that is helpful. the chart stops at the earliest such time anyway, so we might as well say `"Exchange rates are out of date. Last updated {{timeAgo}"`.